### PR TITLE
Added tracking token for Google Analytics

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

Please refer the PR [https://github.com/bastionzero/zli/pull/304]

## Relevant release note information

Release Notes: Added Google Analytics for ZLI

## Related JIRA tickets

Relates to JIRA: CWC-1632

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [ ] No

If yes, please explain:

### CODE REVIEW

File: http/v2/target/types/target.types.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export enum TargetType {
+export enum  TargetType {
```
Comment (tflpd) [inline http/v2/target/types/target.types.ts:1]: extra space

File: http/v2/token/responses/tracking-token-token.responses.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export interface TrackingTokenResponse
```
Comment (tflpd) [inline http/v2/token/responses/tracking-token-token.responses.ts:1]: adjust file name typo

**Review by tflpd — APPROVED** [review]

File: http/v2/token/responses/tracking-token-token.responses.ts (hunk 2 above)
Comment (lipmas) [reply http/v2/token/responses/tracking-token-token.responses.ts:1]: isnt this just a duplicate file..

File: http/v2/target/types/target.types.ts (hunk 1 above)
Comment (SaiprathapReddy7) [reply http/v2/target/types/target.types.ts]: removed extra space

File: http/v2/token/responses/tracking-token-token.responses.ts (hunk 2 above)
Comment (SaiprathapReddy7) [reply http/v2/token/responses/tracking-token-token.responses.ts]: Deleted duplicate one

**Pull request merged**